dpp/src/identity/identity.rs

```diff
@@ -254,15 +228,26 @@ impl Identity {
     }
 
     pub fn from_raw_object(mut raw_object: JsonValue) -> Result<Identity, ProtocolError> {
-        // // TODO identifier_default_deserializer: default deserializer should be changed to bytes
-        // // Identifiers fields should be replaced with the string format to deserialize Identity
         raw_object.replace_identifier_paths(IDENTIFIER_FIELDS_RAW_OBJECT, ReplaceWith::Base58)?;
 
         let identity: Identity = serde_json::from_value(raw_object)?;
 
         Ok(identity)
     }
 
+    pub fn from_raw_identity(raw_object: JsonValue) -> Result<Identity, ProtocolError> {
```

Comment:
Why do we need both?

Reply:
That's an incorrect name - one for where ids and stuff are buffers, and the other one for when they are strings

Reply:
Fixed